src/wdio/services/TerraService.js

```diff
@@ -0,0 +1,53 @@
+import chai from 'chai';
```

Comment:
Lets document each function in AxeService & TerraService so its easy to understand for consumers using these pieces.

Reply:
These shouldn't be documented in the README.

Reply:
Agreed, but for maintainability purposes I think it'd be good to add documentation throughout here as well.

Reply:
Added, sorry I misinterpreted this comment. I agree source should be documented.